src/workloads/scale/LargeIndexedIns.yml

```diff
@@ -15,6 +15,10 @@ Clients:
   Default:
     QueryOptions:
       maxPoolSize: 20000
+  Update:
+    QueryOptions:
+      maxPoolSize: 100
```

Comment:
The CrudActor in this workload had tried to retrieve a client pool named `Update`, but it didn't exist. A side effect of this change is that any undeclared pools cause an error at dry-run evaluation. So I added this one, shouldn't change behavior since `maxPoolSize` defaults to 100 anyway and before it was creating an unconfigured default pool named `Update`. Will confirm in a patch before merging.

Reply:
Will document the need to declare pools explicitly. That behavior has pros and cons, but I'm honestly not sure in this case whether it was intentional or not, so forced explicitness might prevent footgunning.